src/components/sponsers/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { charityAPI } from '../../clients';
+import './styles.css';
+
+class Sponsers extends React.Component {
+  state = { urls: [], urlsError: false };
+  componentDidMount() {
+    charityAPI('/Sponsers')
+      .then(res => {
+        console.log(res);
+        this.setState({
+          urls: res.data
+        });
+      })
+      .catch(error => {});
+  }
+  render() {
+    return this.state.urls.map(({ _id, image }) => (
```

Comment:
This function will work if you received the data you expect but. 
But what **_if data is  still loading_** (We need to tell the user that something is loading) or what if some error occurs (We need to handle that error )
So you need to add some conditional statment
```
    if (//error is true) {
      return <div className="error">error here</div>;
    }

    if (// still loading is true) {
      return <div className="loading">loading .. </div>;
    }
```